estuary_updater/handlers/koji.py

```diff
@@ -96,6 +96,13 @@ def build_handler(self, msg):
                     for component in components:
                         component_build = self.get_or_create_build(component)
                         build.components.connect(component_build)
+            elif build.__label__ == ContainerKojiBuild.__label__:
+                extra_json = msg['body']['msg']['info']['extra']
+                if extra_json.get('typeinfo').get('operator_manifests').get('archive'):
+                    has_operator = True
+                else:
+                    has_operator = False
+                build = ContainerKojiBuild.create_or_update({'operator': has_operator})[0]
```

Comment:
This is creating a new `ContainerKojiBuild` with just the operator property set. This is not what we want and it will fail since some of the required properties are not set.

Reply:
Try looking at this documentation for a better approach:
https://neomodel.readthedocs.io/en/latest/getting_started.html#create-update-delete-operations